jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemView.html

```diff
@@ -18,12 +18,12 @@
     <tbody>
     <tr data-field="root">
 
-        <td class="kie-tablelist__cell">
-            <span data-field="name"></span>
+        <td class="kie-tablelist__cell" style="width:30%">
```

Comment:
missing semicolon _;_ in all _width_ attributes, should be _width:30%;_

Reply:
ok I will change it.